custom_dc/env.list

```diff
@@ -1,7 +1,20 @@
+### API Keys ###
+
 # API key for accessing the base Data Commons API.
 DC_API_KEY=
 # API key for accessing Maps and Places APIs for exploration tools.
 MAPS_API_KEY=
+
+### Directories ###
+
+# Input directory in Google Cloud Storage or local drive, containing files to be processed.
+INPUT_DIR=
+# Output directory in Google Cloud Storage or local drive for generated csv files.
+# Leave blank if same as INPUT_DIR.
```

Comment:
could maybe mark this as an optional param?

Reply:
This one is tricky. This is actually one of the most important variables for the services docker.

Ideally, I would've liked to do `OUTPUT_DIR=$INPUT_DIR` but docker does not apply substitution on these variables. ([ref](https://stackoverflow.com/a/45923906))

I'm wondering if it's best to make it required instead and if both are the same, require that both be set to the same value.

Thoughts?

Reply:
Merging it as-is for now. Can do a follow-up if we decide to make it required.